### ISSUE TITLE
These boxes are not translated (appart from the blue buttons)

### ISSUE THREAD

**Issue by Limezy** [buckets/translations#16]
<img width="885" alt="image" src="https://user-images.githubusercontent.com/24638389/39314930-fadceecc-49a7-11e8-9c81-f62e120c1f48.png">

**Comment by iffy** [comment]
Yes.  I think these are going to be overhauled soon, though, so I'm going to hold off on having you do all the translation work just to throw it away.  Thank you!

**Comment by plangin** [comment]
Any news here? Still not translated (tested in German)
